fieldservice_logbook/views/fsm_equipment_logbook.xml

```diff
@@ -0,0 +1,79 @@
+<odoo>
+
+    <record id="fsm_equipment_logbook_view_search" model="ir.ui.view">
+        <field name="model">fsm.equipment.logbook</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="location_id" />
+                <field name="equipment_id" />
+                <group string="Group By" name="groupby">
+                    <filter
+                        name="location_groupby"
+                        string="Location"
+                        context="{'group_by': 'location_id'}"
+                    />
+                    <filter
+                        name="equipment_groupby"
+                        string="Equipment"
+                        context="{'group_by': 'equipment_id'}"
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="fsm_equipment_logbook_form" model="ir.ui.view">
+        <field name="model">fsm.equipment.logbook</field>
+        <field name="arch" type="xml">
+            <form edit="false" delete="false">
+                <sheet>
+                    <group>
+                        <field name="location_id" />
+                        <field name="equipment_id" />
+                        <field name="type" invisible="1" />
+                        <field
+                            name="source"
```

Comment:
Can't you display the `res_id` (`Many2oneReference`) field directly here, and simply drop this `source` field?

It seems not needed IMO

Reply:
I've tried and it's completely different from usability point of view. Having juts integer on tree view and no reference on form is definitely other functionality.

Reply:
Oh, I thought the `Many2oneReference` used a similar widget by default.
Nevermind then. Thanks for the reply!